### DISCUSSION TITLE
Mac上任意选中文字快速记录到flomo中，可自定义tag（支持多个tag输入）

### DISCUSSION THREAD

**Discussion by southpaw-wu** [flomoapp/3rd-party-tools#30] [分享创造]
https://www.notion.so/southpaw/flomo-mac-577df773cea94698bbf0a1d463d8b36f

**Comment by plidezus** [comment]
赞！

**Comment by bigsongeth** [comment]
👍，解决了我在不同平台收集时的工作流问题。以及，哈哈，这个是隐蔽地输入内容的好方式

**Reply by southpaw-wu** [reply]
确实是，快捷键就能获取输入框，还是很方便的！
